pallets/accounts/src/mock.rs

```diff
@@ -22,7 +38,7 @@ frame_support::construct_runtime!(
 		UncheckedExtrinsic = UncheckedExtrinsic,
 	{
 		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
-		FragPallet: pallet_frag::{Pallet, Call, Storage, Event<T>},
+		AccountsPallet: pallet_accounts::{Pallet, Call, Storage, Event<T>, ValidateUnsigned},
```

Comment:
remove xPallet suffix, it's not used for other

Reply:
done for everything except `Protos`, since there also exists a StorageMap with the name `Protos`